superbench/config/default.yaml

```diff
@@ -19,13 +19,17 @@ superbench:
         - pytorch
     common_model_config: &common_model_config
       duration: 0
-      num_warmup: 16
-      num_steps: 128
+      num_warmup: 64
+      num_steps: 2048
+      sample_count: 8192
+      batch_size: 32
       precision:
         - float32
         - float16
       model_action:
         - train
+      distributed_backend: nccl
+      distributed_impl: ddp
```

Comment:
remove? parameters here are for single process

Reply:
So to run parallel training, we need to add arguments into the CLI command?
Maybe I should add a new config file for validation, since we didn't run model with single GPU for validation.

Reply:
this config section is for single process
to run distributed version, you need to use `sb run` and torch.distributed mode will add these two parameters

Reply:
I see. Removed.

Reply:
we may move this hardcode into mode like mode.parameters in the future